packages/video_player/tizen/src/video_player.cc

```diff
@@ -79,6 +92,35 @@ FlutterDesktopGpuSurfaceDescriptor *VideoPlayer::ObtainGpuSurface(
   return gpu_surface_.get();
 }
 
+void VideoPlayer::InitScreenSaverApi() {
+  screensaver_handle_ = dlopen("libcapi-screensaver.so", RTLD_LAZY);
+  if (!screensaver_handle_) {
+    LOG_INFO("[VideoPlayer] dlopen failed: %s", dlerror());
+    return;
+  }
+
+  screensaver_reset_timeout_ = reinterpret_cast<ScreensaverResetTimeout>(
+      dlsym(screensaver_handle_, "screensaver_reset_timeout"));
+
+  double version = atof(GetPlatformVersion().c_str());
+  if (version > 6.0) {
```

Comment:
Is `screensaver_override_reset` not available on Tizen 6.0 TV? Even so, the version validation might not be necessary since the `dlsym` result is already being checked.

Also what is `screensaver_override_reset`? Is it different from `screensaver_reset_timeout`? (It's poorly documented in PLATZ.)

Reply:
Call `screensaver_override_reset` to override `screensaver_reset` and `screensaver_timeout_reset`.
`screensaver_reset` and `screensaver_timeout_reset` will be ignored, when set `screensaver_override_reset `to 1(true) and work normally when set to 0(false).
You can see the explanation in screensaver.c file.

And `screensaver_override_reset` is only called on Tizen TV 6.5 and above, it is not available on 6.0 and below.

Reply:
Fixed.